resources/assets/js/components/AcceptPayment.vue

```diff
@@ -0,0 +1,170 @@
+                  <li>The check should be signed on the front.</li>
+                  <li>The check should not be endorsed (signed on the back).</li>
+                </ul>
+                <li>Place the check in the cash box.</li>
+                <li>Record the check transaction in the paper receipt book. Offer the carbon copy to the member.</li>
+              </ol>
+            </template>
+            <template v-else-if="payment.method == 'swipe'">
+              <p class="font-weight-bold">Swiped cards incur a $3 processing fee. Add $3 to payment amount.</p>
+              <ol>
+                <li>Open the Square Point of Sale App.</li>
+                <li>Select "Library" from the top tabs.</li>
+                <li>Find the correct dues option and add 1 to the cart</li>
+                <li>Press the charge button, ensuring that the total matches the cost (Plus $3).</li>
+                <li>Follow the prompts to complete the transaction.</li>
+                <li>It is not required to provide a paper receipt for swiped card transactions.</li>
+              </ol>
+            </template>
+            <template v-else-if="payment.method == 'squarecash'">
+              <p class="font-weight-bold">Check with the treasurer before marking SquareCash payments as complete.</p>
+              <ol>
+                <li>View the email saying that the money has been deposited into our account. This is the second email of a given transaction.</li>
+                <li>It is not required to provide a paper receipt for SquareCash transactions.</li>
+              </ol>
+            </template>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-12 col-md-6">
+            <button type="submit" class="btn btn-primary float-right">Submit</button>
+          </div>
+        </div>
+
+      </form>
+    </div>
+  </div>
+</template>
+
+<script>
+  /*
+   *  @props transactionType: Morph of the Transaction model to attach the payment
+   *  @props transactionID: ID of the specific morph of a given Transaction Model
+   *  @props amount: The amount of currency in this payment
+   *  @emit done: Event emitted after a payment is recorded successfully
+   */
+
+  import { numeric, required, sameAs } from 'vuelidate/lib/validators';
+
+  export default {
+    props: {
+      transactionType: {
+        type: String,
+        required: true
+      },
+      transactionId: {
+        type: Number,
+        required: true
+      },
+      amount: {
+        required: true
+      }
+    },
+    data() {
+      return {
+        payment: {
+          payable_id: this.transactionId,
+          payable_type: "App\\" + this.transactionType,
+          method: "",
+          amount: null
+        },
+        paymentMethods: [
+          {value: "cash", text: "Cash"},
+          {value: "check", text: "Check"},
+          {value: "swipe", text: "Swiped Card"},
```

Comment:
Why are we calling this 'swipe'/'Swiped Card' rather than 'Card' or 'Credit/Debit Card'?

Reply:
Because SquareCash is also Debit Card. If we integrate an online cc processor in the future, there may be conflicts. I felt that swiped card was very clear.